src/design_tree_item.cc

```diff
@@ -18,7 +18,7 @@ DesignTreeItem::DesignTreeItem(UHDM::any *item) : item_(item) {
     type_ = "[function]";
   } else if (item_->VpiType() == vpiTask) {
     type_ = "[task]";
-  } else if (item_->VpiType() != vpiModuleInst) {
```

Comment:
I had to make the opposite change a while ago - did the surelog authors change this back recently?